generic/dockerfile/best-practice/use-absolute-workdir.yaml

```diff
@@ -1,12 +1,31 @@
 rules:
   - id: use-absolute-workdir
-    pattern: WORKDIR $WORD
+    pattern-either: 
+      - patterns:
+        - pattern: WORKDIR $VALUE
+        - metavariable-pattern:
+            metavariable: $VALUE
+            patterns:
+              - pattern-not-regex: (\/.*)
+      - patterns:
+        - pattern: 
+            ENV $VAR=$VALUE
+            ...
+            $CMD ${$VAR}
+        - metavariable-pattern:
+            metavariable: $VALUE
+            patterns:
+              - pattern-not-regex: (\/.*)
+        - metavariable-pattern:
+            metavariable: $CMD
+            pattern: WORKDIR
+        - focus-metavariable: $CMD
     message: >-
       Detected a relative WORKDIR. Use absolute paths. This prevents
       issues based on assumptions about the WORKDIR of previous containers.
     severity: WARNING
     languages: [generic]
-    fix: WORKDIR /$WORD
+    #fix: WORKDIR /$VALUE
```

Comment:
intentionally commented-out?

Reply:
Yes, couldn't get the fix to work, so I removed it.